src/dataContext.js

```diff
@@ -1,5 +1,14 @@
-import { createContext } from 'react'
+import React, { useState, createContext } from 'react'
 
 const dataContext = createContext(null)
 
-export default dataContext
+export { dataContext }
+
+export default function DataProvider({ children }) {
```

Comment:
Just for reference, but it's nice if you can encapsulate the provider within a function. Makes it cleaner to use!